source/layouts/layout.html.haml

```diff
@@ -6,6 +6,7 @@
   %meta{name:'description', content: data.page.description}
 
   %title= current_page.data.title || settings.site_name
+  %link{href: 'feed.xml', rel: 'alternate', title: 'RSS', type: 'application/rss+xml'}
```

Comment:
Line is too long. [86/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace